### PR TITLE
Customer payment integration

### PR DESCRIPTION
**Pull request by ifebrand6**
You will need to run `bundle exec Figaro install ` and update the _application.yml_ files for this to work.

### CODE REVIEW

**Comment by ifebrand6** [comment]
I cherry-pick the commit from a conflicted branch, hence the many commits

File: app/assets/javascripts/customer_dashboard.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (Innarticles) [inline app/assets/javascripts/customer_dashboard.coffee:3]: Remove file

File: app/assets/stylesheets/customer_dashboard.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the CustomerDashboard controller here.
+// They will automatically be included in application.css.
```
Comment (Innarticles) [inline app/assets/stylesheets/customer_dashboard.scss:2]: Remove empty file

File: test/controllers/customer_dashboard_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class CustomerDashboardControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
```
Comment (Innarticles) [inline test/controllers/customer_dashboard_controller_test.rb:4]: remove test folder

**Pull request merged**